client/components/token-field/style.scss

```diff
@@ -57,6 +57,18 @@ input[type="text"].token-field__input {
 	padding: 0;
 	color: $white;
 	overflow: hidden;
+
+	&.is-success {
+		.token-field__token-text, .token-field__remove-token {
```

Comment:
No, because the `is-success` or `is-error` is currently applied to the parent of the token component: `.token-field__token`.

The selector you suggested expect that the `is-success` and `is-error` are both on the token text as well as the remove token (the "x").

We could potentially change where the modifier classes are added though.